### DISCUSSION TITLE
Request: Separate playlist window from Viewer

### DISCUSSION THREAD

**Discussion by fredfx1957** [videovillage/community#121] [Ideas]
It would be great if you could separate the playlist window from the viewer window. That way, in a two monitor set up you could have the viewer full screen in one window and the playlist in another window.

**Comment by gkertai** [comment]
+1
I agree, this would be lovely when I need to review footage in full-screen. 

I suppose this is not so much about detaching the playlist, but rather allowing video output to go to another monitor while retaining all controls (timeline, playlist, markers, etc.) on another monitor. Could this be implemented in a way similar to the "A/V Output" mode in Final Cut Pro?
